app/js/services.js

```diff
@@ -189,6 +189,14 @@ angular.module('myApp.services', ['myApp.i18n', 'izhukov.utils'])
       return getUser(myID)
     }
 
+    function getSavedMessages () {      
+      var apiUser = jQuery.extend(true, {}, getUser(myID))
+        apiUser.rFirstName = _('user_first_name_self')
+        apiUser.rFullName = _('user_name_self')
+        apiUser.photo._ = 'self'
```

Comment:
.photo might be missing

Also, when user lacks photo, saved messages icon gets replaced by initials sometimes